Resources/doc/emails.rst

```diff
@@ -24,6 +24,27 @@ To enable it, update your configuration as follows:
             confirmation:
                 enabled: true
 
+Confirmation of Changed Email
+-----------------------------
+
+When a user changes her email address on the edit profile page,
```

Comment:
an user

Reply:
actually "a user" is correct as "user" is pronounced "yuser" and before a "y" the "a" us used instead of "an". See a-user-or-an-user question on stackexchange and many other sites for reference.